packages/ui/src/v2/components/ct-code-editor/ct-code-editor.ts

```diff
@@ -436,9 +441,11 @@ export class CTCodeEditor extends BaseElement {
       if (charm) {
         // this is VERY specific
         // if you do `getEntityId(mentionableArray.key(i))` you'll get a different answer (the ID of the array itself)
-        const charmIdObj = getEntityId(mentionableArray.get()[i]);
-        const charmId = charmIdObj?.["/"] || "";
-        if (charmId === id) {
+        const charmIdObjA = getEntityId(mentionableArray.get()[i]);
+        const charmIdObjB = getEntityId(mentionableArray.key(i));
```

Comment:
I have to double check, but I think that getEntityId ignores to path part, so all of these IDs will be the same irrespective of `i`.